pkg/debug/debug.go

```diff
@@ -0,0 +1,24 @@
+package debug
+
+import (
+	"log/slog"
+	"os"
+)
+
+func LogInfo(msg string, args ...any) {
+	if isDebugMode {
+		slog.Info(msg, args...)
```

Comment:
Since it's debug mode, should we log to debug instead? :)

Reply:
Yeah, good point. I was even thinking to change it to "log level", but let's for now make "debug_mode = true" => "log_level = debug".

Reply:
I just realized that there is a small problem with this. The default level is `info` and we probably don't want to change it for the global logger.